addons/sale_coupon/models/sale_coupon.py

```diff
@@ -108,3 +108,14 @@ def action_coupon_sent(self):
             'target': 'new',
             'context': ctx,
         }
+
+    def cron_expire_coupon(self):
+        self._cr.execute("""
+            SELECT C.id FROM SALE_COUPON as C
+            INNER JOIN SALE_COUPON_PROGRAM as P ON C.program_id = P.id
+            WHERE C.STATE in ('reserved', 'new')
```

Comment:
why is `sent` removed here? in comparaison with the original commit https://github.com/odoo/odoo/pull/34620/commits/7092d0ae9b02a2b12d953e3d0bc3cc77fa6df5e1#diff-71f27398472ab8ae3d7b87a74c87004bd17adea6922e27ab4cf8e2bcd98ff60aR125

Reply:
Because `sent` state doesn't exist in version 13:
https://github.com/odoo/odoo/blob/13.0/addons/sale_coupon/models/sale_coupon.py#L29
Contrary at the version 14:
https://github.com/odoo/odoo/blob/14.0/addons/coupon/models/coupon.py#L29